linebot/send_message.go

```diff
@@ -292,7 +293,21 @@ func (call *NarrowcastCall) WithDemographic(demographic DemographicFilter) *Narr
 
 // WithLimitMax method will set maximum number of recipients
 func (call *NarrowcastCall) WithLimitMax(max int) *NarrowcastCall {
-	call.limit = &NarrowcastMessageLimit{Max: max}
+	if call.limit == nil {
```

Comment:
If we check call.limit here, we'll also need to check call.filter? It's better to keep it simple.